<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDialog</class>
 <widget class="QDialog" name="DeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="PrimaryKey">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="PrimaryKeyValue">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PrimaryKey</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PrimaryKeyValue</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Deleterow">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete row</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
